sdk/core/System.ServiceModel.Rest/api/System.ServiceModel.Rest.net6.0.cs

```diff
@@ -9,12 +9,10 @@ public enum ErrorBehavior
     public partial class InvocationOptions
     {
         public InvocationOptions() { }
-        public virtual bool BufferResponse { get { throw null; } set { } }
```

Comment:
This is now set using a static method on the ResponseBufferingPolicy.

Reply:
Actually, I imagine we'll want to move this to the transport, because why does it make sense to turn off response buffering on a response buffering policy?  Shouldn't we just leave it out of the pipeline if we don't want to buffer the response?

But we can't do that today, because of the network timeout functionality and the read timeout stream being all coupled to this policy.